test/lit/passes/coalesce-locals-eh.wast

```diff
@@ -0,0 +1,86 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --coalesce-locals -all -S -o - | filecheck %s
+
+(module
+  ;; CHECK:      (tag $e)
+
+  ;; CHECK:      (tag $any (param (ref any)))
+
+  ;; CHECK:      (func $bar (type $2) (result i32)
+  ;; CHECK-NEXT:  (i32.const 1984)
+  ;; CHECK-NEXT: )
+  (func $bar (result i32)
+    (i32.const 1984)
+  )
+
+ (tag $e)
+
+ (tag $any (param (ref any)))
```

Comment:
Indentation here looks different than the function before. Also, please put the tags first, before functions.